src/Components/RegisterForm.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+namespace ARKEcosystem\Fortify\Components;
+
+use ARKEcosystem\Fortify\Components\Concerns\ValidatesPassword;
+use ARKEcosystem\Fortify\Models;
+use Domain\Collaborator\Models\Invitation;
+use Livewire\Component;
+
+class RegisterForm extends Component
```

Comment:
Do we really need the whole register process to be a Livewire component or would it be sufficient to have the password part to be Livewire?

Reply:
@faustbrian The first version of this feature was a standalone livewire component just for the password. The problem was that the update password form is another livewire component, and the login form was a regular blade view. I dealt with different problems related to validation messages, live updates, etc., between both forms.

After a few tests, this ends being the simplest and reusable solution. Plus, I put the form logic that used to be in the service provider encapsulated inside this component.